raft/bin/test-kraft-server-start.sh

```diff
@@ -17,7 +17,7 @@
 base_dir=$(dirname $0)
 
 if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
-    export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/raft-log4j.properties"
+    export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/kraft-log4j.properties"
```

Comment:
I like that we are renaming a few of the files in the module (`raft`). How far do we want to take this? For example:
1. Should we rename the `raft` module to `kraft`?
2. Should we rename the `org.apache.kafka.raft` to `org.apache.kafka.kraft`? Maybe this is okay since `kafka` precedes `raft`.

We don't have to do this in this PR. I am just interested to know if we are only concerned with public facing names or should we also rename internal names?

cc @hachikuji

Reply:
@jsancio The folder names for the modules don't `kafka` even though the published artifacts do (`clients` and `kafka-clients`). The artifact for the `raft` module is `kafka-raft` and it was originally released in 2.7. I don't want to change that right now. Similarly, for the package names, since they're `kafka.raft`, I think we can leave it like that.

Reply:
Good to know about the artifact names. Sounds good to me.